config/httpserver.go

```diff
@@ -0,0 +1,34 @@
+package config
+
+import (
+	"github.com/multiformats/go-multiaddr"
+	manet "github.com/multiformats/go-multiaddr/net"
+)
+
+type HttpServer struct {
+	Enabled         bool
```

Comment:
Please document behaviour of this flag; enabling HTTP publisher seems to disable dtsync publisher?

Reply:
Changed via enum: 0a8a8bb1697d77109f30f20627fc034e77da98f4